front/src/views/user/user-settings/user-settings.view.js

```diff
@@ -1,19 +1,40 @@
 import { MmtTextInput } from '@components';
 import { mapState } from 'vuex';
+import { VFileInput, VIcon } from 'vuetify/lib';
 const { emailRegex } = require('@/config/constants.json');
 
 export default {
   components: {
-    MmtTextInput
+    MmtTextInput,
+    VFileInput,
+    VIcon
   },
   data () {
     return {
       editedUserSettings: {},
       formValidity: true,
       emailIsAlreadyTaken: false,
+      fileUploadError: '',
       emailRules: [
         email => RegExp(emailRegex).test(email)
         || this.$t('userSettings.enterCorrectEmail')
+      ],
+      avatarRules: [
+        value => {
+          if (value && value.size > 1024 * 1024 * 2) {
+            return this.$t('userSettings.fileIsToBig');
+          }
```

Comment:
const MAX_AVATAR_SIZE_MB = 2;
const MAX_AVATAR_SIZE_KB = 1024 * 1024 * MAX_AVATAR_SIZE_MB;
....
avatarRules: [
        value => !value || value.size < MAX_AVATAR_SIZE_KB 
            || `${this.$t('userSettings.fileIsToBig')}'. `${this.$t('common.correctFileSize')}: ${MAX_AVATAR_SIZE_MB} mb. ,
      ],

Reply:
thank, fixed rules